lib/media.js

```diff
@@ -203,22 +202,27 @@ function flatMap(list, fn) {
 }
 
 /**
- * Get the proper styles for the component
+ * Get the proper styles for the component. If a styleguide is configured, use
+ * that css. Defaults to the site slug
  *
  * @param {string} name
  * @param {object} site
  * @returns {array}
  */
 function getStyles(name, site) {
-  const slug = site.slug,
+  const stylesSource = site.styleguide || site.slug,
     assetDir = site && site.assetDir || MEDIA_DIRECTORY,
     assetPath = site && site.assetPath || '',
-    assetHost = site && site.assetHost || '';
+    assetHost = site && site.assetHost || '',
+    filenamesFilter = [
```

Comment:
Don't really understand this variable name — this array isn't used to filter anything, though items are filtered out of it later. Maybe simply `cssFilePaths`?